packages/mobile-top-nav/src/media-menu.js

```diff
@@ -0,0 +1,153 @@
+import { LitElement, html } from 'lit-element';
+
+import './media-slider';
+import './assets/img/media-menu-images';
+import mediaMenuCss from './css/media-menu';
+
+const menuSelection = [
+  {
+    icon: 'web',
+    menu: 'web',
+    label: 'Wayback Machine',
+  },
+  {
+    icon: 'texts',
+    menu: 'texts',
+    label: 'Texts',
+  },
+  {
+    icon: 'video',
+    menu: 'video',
+    label: 'Video',
+  },
+  {
+    icon: 'audio',
+    menu: 'audio',
+    label: 'Audio',
+  },
+  {
+    icon: 'software',
+    menu: 'software',
+    label: 'Software',
+  },
+  {
+    icon: 'images',
+    menu: 'images',
+    label: 'Images',
+  },
+  {
+    icon: 'ellipses',
+    menu: 'more',
+    label: 'More',
+  },
+];
+
+class MediaMenu extends LitElement {
+  static get properties() {
+    return {
+      mediaMenuOpen: { type: Boolean },
+      mediaMenuAnimate: { type: Boolean },
+      mediaSliderOpen: { type: Boolean },
+      mediaSliderAnimate: { type: Boolean },
+      selectedMenuOption: { type: String },
+    };
+  }
+
+  constructor() {
+    super();
+    this.mediaSliderOpen = false;
+    this.mediaSliderAnimate = false;
+    this.selectedMenuOption = '';
+  }
+
+  updated(changedProperties) {
+    const { mediaMenuOpen, mediaSliderOpen } = this;
+    const menuClosed = changedProperties.has('mediaMenuOpen')
+      && changedProperties.get('mediaMenuOpen')
```

Comment:
`changedProperties.get('mediaMenuOpen')` should be the same as `this.mediaMenuOpen` here.

Reply:
If `mediaMenuOpen` is the property that triggered the lifecycle event, it is not the same as the current property value. https://lit-element.polymer-project.org/guide/lifecycle#updated

Reply:
Oh are you looking for the previous value of `mediaMenuOpen`?

Reply:
Yeah, since this fires when any property changes, there's a check for whether this change is due to the media slider closing. To do that, we have to first check whether the `mediaMenuOpen` property is changing, and then check to see if it was previously open. If so, the currently opened slider menu needs to close as well.